docs/topics/lombok.md

```diff
@@ -0,0 +1,137 @@
+[//]: # (title: Lombok compiler plugin)
+
+> The Lombok compiler plugin is [Experimental](components-stability.md). 
+> It may be dropped or changed at any time. Use it only for evaluation purposes. 
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-7112).
+>
+{type="warning"}
+
+The Lombok compiler plugin allows the generation and use of Java's Lombok declarations within the same mixed Java/Kotlin module.
+If you generate these declarations in a Java module and use them in another Kotlin module, 
```

Comment:
The sentence needs a couple of improvements to be strictly true:
- "If you generate" - not necessarily "you", it can be anyone. Author of a library you use, for example.
- "in a Java module" - what if this module also contains code in Kotlin or other JVM languages?

Reply:
To avoid passive, you can start from "If ... declarations come from .."

Reply:
>"in a Java module" - what if this module also contains code in Kotlin or other JVM languages?

Don't know anything about other JVM languages, do we consider any mixes in our docs?

>what if this module also contains code in Kotlin

Isn't "a Java module" enough to define that this module contains only code in Java? I'll add "a pure Java module".

>Author of a library you use, for example.

Here we're talking about code generation from sources and some peculiarities of processing such code with annotation processors. When using libraries, we use already generated code (re-asked this from the developer and he's agreed).